pkg/client/interface.go

```diff
@@ -1,4 +1,4 @@
-//go:generate mockgen -destination=../../internal/mocks/mockclient/events_query_client_mock.go -package=mockclient . Dialer,Connection
+//go:generate mockgen -destination=../../internal/mocks/mockclient/events_query_client_mock.go -package=mockclient . Dialer,Connection,EventsQueryClient
```

Comment:
Seeing we have different packages but keep them defined in `interface.go`, it'll be really nice to get a sense of all the different interfaces we have available.

In fact, this is how I went down the #128 🐇 hole